trusted.ci.jenkins.io.tf

```
@@ -189,6 +201,13 @@ resource "azurerm_virtual_machine_data_disk_attachment" "trusted_ci_controller_d
   caching            = "ReadWrite"
 }
 
+resource "azurerm_private_dns_a_record" "trusted_ci_controller" {
```

```suggestion
resource "azurerm_private_dns_a_record" "trusted_ci_controller_ip" {
```
(not sure about this one)

---

+1 with @timja